app/services/flow/base_flow.rb

```diff
@@ -45,11 +45,8 @@ def form_response(obj, value)
       response
     end
 
-    # Duck type check for FormResponse
     def form_response?(response)
-      response.is_a?(FormResponse) ||
-        response.is_a?(Acuant::Response) ||
-        (response.respond_to?(:success?) && response.respond_to?(:to_h))
+      response.is_a?(FormResponse) || response.is_a?(DocAuthClient::Response)
```

Comment:
![duck hunt video game box cover](https://upload.wikimedia.org/wikipedia/en/1/14/DuckHuntBox.jpg)

Reply:
Maybe for a separate refactor, I would love to update to have this *enforce* all responses be one of those